<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmSearchReplace</class>
 <widget class="QDialog" name="frmSearchReplace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/notepadqq.svg</normaloff>:/icons/icons/notepadqq.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabSearch">
      <attribute name="title">
       <string>Search</string>
      </attribute>
      <widget class="QPushButton" name="btnFindPrev">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>20</y>
         <width>31</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>⇧</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>25</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnFindNext">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>20</y>
         <width>31</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>⇩</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbSearch">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>261</width>
         <height>27</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSelectAll">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>120</y>
         <width>131</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Select all</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabReplace">
      <attribute name="title">
       <string>Replace</string>
      </attribute>
      <widget class="QComboBox" name="cmbReplace">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>55</y>
         <width>261</width>
         <height>27</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Replace with</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>25</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnFindNext_3">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>20</y>
         <width>31</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>⇩</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbSearch_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>261</width>
         <height>27</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btnFindPrev_3">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>20</y>
         <width>31</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>⇧</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btnReplaceNext">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>120</y>
         <width>111</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Replace ⇩</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btnReplacePrev">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>120</y>
         <width>111</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Replace ⇧</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btnReplaceAll">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>120</y>
         <width>131</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Replace all</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
